### PR TITLE
526 contact info content updates

### PR DESCRIPTION
**Pull request by bshyong**
## Description
Added content updates to contact info section

## Testing done
Local testing

## Screenshots
screen shot 2018-12-10 at 5 23 27 pm

screen shot 2018-12-10 at 5 23 31 pm

## Acceptance criteria
- [x] Requirements in https://github.com/department-of-veterans-affairs/vets.gov-team/issues/15441

## Definition of done
- [x] Events are logged appropriately
- [x] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

Resolves https://github.com/department-of-veterans-affairs/vets.gov-team/issues/15441

### CODE REVIEW

File: src/applications/disability-benefits/all-claims/components/ReviewCardField.jsx [hunk 1]
```diff
@@ -151,11 +158,13 @@ export default class ReviewCardField extends React.Component {
 
     const { volatileData, editTitle } = this.props.uiSchema['ui:options'];
     const title = editTitle || this.getTitle();
+    const subtitle = this.getSubtitle();
 
     return (
       <div className="review-card">
         <div className="review-card--body input-section va-growable-background">
           <h4 className="review-card--title">{title}</h4>
+          <div>{subtitle}</div>
```
Comment (cvalarida) [inline src/applications/disability-benefits/all-claims/components/ReviewCardField.jsx]: What do you think of making this block conditional?
```jsx
{subtitle && <div>{subtitle}</div>}
```

File: src/applications/disability-benefits/all-claims/pages/contactInformation.js [hunk 2]
```diff
@@ -9,18 +9,19 @@ import PhoneNumberWidget from 'us-forms-system/lib/js/widgets/PhoneNumberWidget'
...
+  PrimaryAddressViewField,
 } from '../content/contactInformation';
 
 import {
-  validateZIP,
+  isInFuture,
   validateMilitaryCity,
   validateMilitaryState,
-  isInFuture,
+  validateZIP,
```
Comment (cvalarida) [inline src/applications/disability-benefits/all-claims/pages/contactInformation.js]: Heh, out of curiosity, do you have an import alphabetizer plugin for your editor or something? :smile:

File: src/applications/disability-benefits/all-claims/components/ReviewCardField.jsx [hunk 3]
```diff
@@ -108,6 +108,13 @@ export default class ReviewCardField extends React.Component {
       : uiSchema['ui:title'];
   };
 
+  getSubtitle = () => {
```
Comment (cvalarida) [inline src/applications/disability-benefits/all-claims/components/ReviewCardField.jsx]: I think I like this. Can you add a test or two for it?

**Review by cvalarida — COMMENTED** [review]
Looks good! I think the new `ReviewCardField` functionality could use some testing, but it's golden otherwise.

File: src/applications/disability-benefits/all-claims/components/ReviewCardField.jsx (hunk 3 above)
Comment (bshyong) [reply src/applications/disability-benefits/all-claims/components/ReviewCardField.jsx]: Will do

File: src/applications/disability-benefits/all-claims/components/ReviewCardField.jsx (hunk 1 above)
Comment (bshyong) [reply src/applications/disability-benefits/all-claims/components/ReviewCardField.jsx]: Yep, agree

File: src/applications/disability-benefits/all-claims/pages/contactInformation.js (hunk 2 above)
Comment (bshyong) [reply src/applications/disability-benefits/all-claims/pages/contactInformation.js]: I have a plugin but it's manually activated—I sorted these to make it easier to scan 🔡

**Comment by bshyong** [comment]
Ready for re-review

**Review by markgreenburg — APPROVED** [review]
lgtm!

**Pull request merged**